src/protocol/bolt/src/main/java/org/apache/jmeter/protocol/bolt/sampler/BoltSampler.java

```diff
@@ -82,7 +83,8 @@ public SampleResult sample(Entry e) {
 
         try {
             res.setResponseHeaders("Cypher request: " + getCypher());
-            res.setResponseData(execute(BoltConnectionElement.getDriver(), getCypher(), params), StandardCharsets.UTF_8.name());
+            res.setResponseData(execute(BoltConnectionElement.getDriver(), BoltConnectionElement.getDatabaseName(),
```

Comment:
What about having a session config in the `BoltConnectionElement` ? The database name and potentially other config options would be best controlled there and adding additional config options would not require changing the sampler again